my-app/src/app/courses-page/course-search/course-search.component.css

```diff
@@ -0,0 +1,54 @@
+:host {
+    margin: 20px auto 0 15%;
+    display: flex;
+  }
+
+.course-search {
+  border-radius: 3px;
+  border: 1px solid var(--borderGray);
+  box-sizing: border-box;
+  height: 36px;
+  width: 270px;
+  margin-right: 12px;
+  padding: 9px 12px 9px 40px;
+}
+
+
+.search-button {
+  line-height: 30px;
+  letter-spacing: -0.08px;
+  color: var(--white);
+  font-size: 14px;
+  font-family: var(--my-font-family);
+  text-transform: none;
+  font-weight: 600;
+  border-radius: 3px;
+  background: var(--lightGreen);
+  padding: 3px 17px;
+  border: none;
+}
+
+.course-search:placeholder-shown {
+  background-image: url('../../../assets/icons/magnifying_glass.svg');
```

Comment:
if you move this component into another place this will break. Try to use an absolute path for such things
`background-image: url('src/assets/icons/magnifying_glass.svg');`